_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,64 @@
 title: Banner Component
 ---
 
-The purpose of the Banner component is to render a media in a given page slot. The media can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
+The banner component is used to render different banners that are created in the CMS. Banners contain one or multiple images, as well as some optional content, such as a header. A banner is often used to link to other content.
 
 ## CMS Component Binding
 
-There are multipe banner component types in the CMS:
+There are multiple banner component types in the CMS, as follows:
 
--   `BannerComponent`
--   `SimpleBannerComponent`
--   `SimpleResponsiveBannerComponent`
+- `BannerComponent`
+- `SimpleBannerComponent`
+- `SimpleResponsiveBannerComponent`
 
-There is also a `RotatingImagesComponent` which renders a one or multiple `BannerComponents` in a carousel.
+There is also a `RotatingImagesComponent`, which is a so-called container component. A container component holds multiple components. The `RotatingImagesComponent` is used to render multiple `BannerComponent` components in a carousel.
 
-The different types have been created for historical reasons, there's not much of a point to distinquish them anymore; there's only component implementation in Spartacus for banners. Sparatacus always renders banners using adaptive and responsive features to render the most optimal image for the given device.
+The different banner component types have been created for historical reasons, but there is not much to distinguish them anymore: in Spartacus, there is only component implementation for banners. Spartacus renders the banner media using the media component, which includes lazy loading and loading specific media for specific dimensions. For more information, see [Media Component]({{ site.baseurl }}{% link _pages/dev/components/shared-components/media-component.md %}).
 
-The CMS Banner component is mapped to the Spartacus implementation as follows:
+The CMS banner component is mapped to the Spartacus implementation as follows:
 
 ```typescript
 <CmsConfig>{
-    SimpleResponsiveBannerComponent: {
-        component: BannerComponent
-    },
-    BannerComponent: {
-        component: BannerComponent
-    },
-    SimpleBannerComponent: {
-        component: BannerComponent
-    }
+  SimpleResponsiveBannerComponent: {
+    component: BannerComponent,
+  },
+  BannerComponent: {
+    component: BannerComponent,
+  },
+  SimpleBannerComponent: {
+    component: BannerComponent,
+  },
```

Comment:
I'd drop the leading comma's here. We use those leading commas in our code to easier compare on githubs code comparison tooling, but in docs it looks weird.

FYI: As a rule of thumb, the last property of a json object should not have a comma. Only to separate properties, you'd use a coma. In the code above, the right readable syntax would be:
```ts
<CmsConfig>{
  cmsComponents: {
    SimpleResponsiveBannerComponent: {
      component: BannerComponent
    },
    BannerComponent: {
      component: BannerComponent
    },
    SimpleBannerComponent: {
      component: BannerComponent
    }
  }
}
```

Reply:
Thank you for explaining those details 🙂 
I'll just point out that when it comes to code examples, I try to read them, of course, to make sure I understand what the doc is explaining, but if I think something needs changing, I always verify with the developer first. So in the case of all these commas being added, perhaps it was an extension in your IDE that added them? Lint? I wouldn't dream of adding commas to code examples on my own 😄